includes/abstracts/llms-abstract-admin-wizard.php

```diff
@@ -230,7 +230,8 @@ public function get_prev_step( string $step = null ) {
 	 */
 	private function get_save_text( string $step ): string {
 
-		$default = esc_html__( 'Save & Continue', 'lifterlms' );
+		// `html_entity_decode()` needed for unit tests to pass.
+		$default = html_entity_decode( esc_html__( 'Save & Continue', 'lifterlms' ) );
```

Comment:
@seothemes the test is failing because it expects `Save & Continue`, that's because before we didn't escape the localized  default text. If we want to keep on escaping at this stage (but we could just do that when printing the string, that's when is really required, but I'm fine with escape as soon as you can), I don't really see why we should  apply `html_entity_decode` to this string and not to all the other steps' ones.
Now honestly I don't think we should do you?
If we want keep on escaping the output here, let's just adapt the tests :D